lib/oga/xml/document.rb

```diff
@@ -57,6 +57,12 @@ def root_node
         self
       end
 
+      # Returns the top level element in the document.
+
```

Comment:
Please remove the empty line here and adjust the documentation as follows:

```
# Returns the first Element in the document
#
# @return [Oga::XML::Element]
```

Right now the documentation is not entirely correct. While in most cases the first element will be the top-level element there's nothing preventing there from being multiple top-level elements (code wise). As such it's better to state that this method simply returns the first Element it finds.